opba-api-security/src/main/java/de/adorsys/opba/api/security/external/domain/OperationType.java

```diff
@@ -13,4 +13,12 @@ public static boolean isTransactionsPath(String path) {
     public static boolean isBankSearchPath(String path) {
         return path.contains("/bank-search");
     }
+
+    public static boolean isGetPaymentStatus(String path) {
+        return path.contains("/status");
+    }
+
+    public static boolean isGetPayment(String method) {
+        return "GET".equals(method);
```

Comment:
"GET" should be some enum, or String constant

Reply:
Enum is used instead